solutions/devsprint-riccardo-1/OnboardingChallenge/Protocols/ViewCode.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ViewCode.swift
+//  OnboardingChallenge
+//
+//  Created by Pedro Gabriel on 15/12/21.
+//
+
+import UIKit
+
+protocol ViewCode where Self: UIView {
+    func configureSubviews()
+    func configureSubviewsConstraints()
+    func configureAdditionalBehaviors()
```

Comment:
What do you expect should go in the `configureAdditionalBehaviors` method?

Reply:
This configureAdditionalBehaviors is used to configure things like background, font, or something else that is not addSubviews, or constraints.

Reply:
Got it. What could be confusing is that `behaviors` usually means functionalities, like button taps or gestures. In this case, you'd like to wrap some static styling in the `configureAdditionalBehaviors`. For the future, my suggestion is to use more informative names, like `style` and `configureGestures` or `configureAnimations`. In this way, teammates will immediately know what the method will do! 😉